android/app/src/main/java/app/organicmaps/car/util/IntentUtils.java

```diff
@@ -39,6 +41,15 @@ else if (Intent.ACTION_VIEW.equals(action))
       processViewIntent(carContext, intent);
   }
 
+  @NonNull
+  public static PendingIntent createSearchIntent(@NonNull CarContext context, @NonNull String query)
+  {
+    final String uri = "geo:0,0?q=" + query.replace(" ", "+");
```

Comment:
Why geo? We have `om://search` api that is way more flexible: https://omaps.app/api

Reply:
We're in AA world, it supports only `geo` intents.
Yes, in this scenario, we can utilize `om://search` but I don't see any particular reason for doing so.

Reply:
@AndrewShkrob can you please check this case and create an issue, if necessary? If other apps are opened instead of OM, that will break the use-case for our users. We should know about it and fix it.

@rtsisyk why merge the code with unresolved comments?

Reply:
> Is geo: link always opened inside our app? What if there are any other apps installed?

Regarding this case (and any other case related to AA), yes

Technically, it's not possible to choose the default navigation app. AA will pass intent with `geo:` link to the last opened navigation app. If you were using Google Maps, then AA will will redirect the intent from the other AA app to the Google Maps. If you were using OM, then to the OM.

Reply:
@AndrewShkrob is there a way to open om:// links/intents or https links listed at https://omaps.app/api ?